pkg/controller/argocdmetrics/argocd_metrics_controller_test.go

```diff
@@ -70,7 +72,7 @@ func TestReconcile_add_namespace_label(t *testing.T) {
 		namespace    string
 	}{
 		{
-			instanceName: "argocd-cluster",
+			instanceName: common.ArgoCDInstanceName,
```

Comment:
In tests, we may not want to use constants (that are shared with production code) so we can catch typo bug or unexpected change in contant values.